autoendpoint/src/extractors/subscription.rs

```diff
@@ -298,7 +326,12 @@ fn validate_vapid_jwt(
                 return Err(VapidError::InvalidVapid(e.to_string()).into());
             }
             _ => {
-                metrics.clone().incr("notification.auth.bad_vapid.other");
+                let mut tags = Tags::default();
+                tags.tags.insert("error".to_owned(), e.to_string());
```

Comment:
This isn't a suitable tag value, lots of their [error variants produce strings with whitespace](https://docs.rs/jsonwebtoken/latest/src/jsonwebtoken/errors.rs.html#110)